libfreeipmi/include/freeipmi/spec/ipmi-ipv6-address-status-spec.h

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Perhaps I did not choose .h files best in the beginning, but since the other macros for other entries in the lan config table are in ipmi-lan-cmds.h, lets stick it in there for now.